odoo/models.py

```diff
@@ -4136,9 +4136,11 @@ def _create(self, data_list):
         return records
 
     def _compute_field_value(self, field):
+        if field.async_compute and not self.env.context.get('async_compute'):
+            self[field.name] = False
```

Comment:
Why don't you override the problematic model with a specific hack? (`if field.name == ... and ...`)
Introducing this at the heart of the ORM looks like a very bad idea to me.

Reply:
Because Odoo could use more async IMO; not only here.
Maybe I'm just looking too far...